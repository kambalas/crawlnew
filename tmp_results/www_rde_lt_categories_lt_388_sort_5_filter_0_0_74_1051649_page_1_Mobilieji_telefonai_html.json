[
  {
    "name": "iPhone 15 Plus 128GB Black",
    "price": "849.30",
    "error": false
  },
  {
    "name": "iPhone 15 Plus 128GB Green",
    "price": "886.81",
    "error": false
  },
  {
    "name": "iPhone 15 Plus 128GB Blue",
    "price": "889.00",
    "error": false
  },
  {
    "name": "iPhone 15 Plus 256GB Black",
    "price": "953.11",
    "error": false
  },
  {
    "name": "iPhone 15 Plus 256GB Pink",
    "price": "1073.11",
    "error": false
  },
  {
    "name": "iPhone 15 Plus 512GB Pink",
    "price": "1252.28",
    "error": false
  },
  {
    "name": "iPhone 15 Plus 512GB Yellow",
    "price": "1252.28",
    "error": false
  },
  {
    "name": "iPhone 15 Plus 512GB Green",
    "price": "1254.59",
    "error": false
  },
  {
    "name": "iPhone 15 Plus 512GB Blue",
    "price": "1266.90",
    "error": false
  },
  {
    "name": "iPhone 15 Plus 512GB Black",
    "price": "1282.86",
    "error": false
  },
  {
    "name": "iPhone 15 Plus 256GB Yellow",
    "price": "1427.08",
    "error": false
  },
  {
    "name": "iPhone 11",
    "price": "",
    "error": false
  },
  {
    "name": "iPhone 12",
    "price": "",
    "error": false
  },
  {
    "name": "iPhone 13",
    "price": "",
    "error": false
  },
  {
    "name": "iPhone 14",
    "price": "",
    "error": false
  },
  {
    "name": "iPhone 14 Plus",
    "price": "",
    "error": false
  },
  {
    "name": "iPhone 14 Pro Max",
    "price": "",
    "error": false
  },
  {
    "name": "iPhone 15",
    "price": "",
    "error": false
  },
  {
    "name": "iPhone 15 Plus",
    "price": "",
    "error": false
  },
  {
    "name": "iPhone 15 Pro",
    "price": "",
    "error": false
  },
  {
    "name": "iPhone 15 Pro Max",
    "price": "",
    "error": false
  },
  {
    "name": "iPhone 16",
    "price": "",
    "error": false
  },
  {
    "name": "iPhone 16 Plus",
    "price": "",
    "error": false
  },
  {
    "name": "iPhone 16 Pro",
    "price": "",
    "error": false
  },
  {
    "name": "iPhone 16 Pro Max",
    "price": "",
    "error": false
  },
  {
    "name": "iPhone SE 4",
    "price": "",
    "error": false
  },
  {
    "name": "iPhone SE",
    "price": "",
    "error": false
  }
]