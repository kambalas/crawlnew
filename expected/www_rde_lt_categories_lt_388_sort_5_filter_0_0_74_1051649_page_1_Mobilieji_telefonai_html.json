[
  {
    "index": 0,
    "error": true,
    "tags": [
      "error"
    ],
    "content": "'str' object has no attribute 'choices'"
  },
  {
    "index": 3,
    "error": true,
    "tags": [
      "error"
    ],
    "content": "'list' object has no attribute 'usage'"
  },
  {
    "name": "iPhone 13 128GB Midnight",
    "price": "514.96",
    "error": false
  },
  {
    "name": "iPhone 11",
    "price": "",
    "error": false
  },
  {
    "name": "iPhone 12",
    "price": "",
    "error": false
  },
  {
    "name": "iPhone 13",
    "price": "",
    "error": false
  },
  {
    "name": "iPhone 14",
    "price": "",
    "error": false
  },
  {
    "name": "iPhone 14 Plus",
    "price": "",
    "error": false
  },
  {
    "name": "iPhone 14 Pro Max",
    "price": "",
    "error": false
  },
  {
    "name": "iPhone 15",
    "price": "",
    "error": false
  },
  {
    "name": "iPhone 15 Plus",
    "price": "",
    "error": false
  },
  {
    "name": "iPhone 15 Pro",
    "price": "",
    "error": false
  },
  {
    "name": "iPhone 15 Pro Max",
    "price": "",
    "error": false
  },
  {
    "name": "iPhone 16",
    "price": "",
    "error": false
  },
  {
    "name": "iPhone 16 Plus",
    "price": "",
    "error": false
  },
  {
    "name": "iPhone 16 Pro",
    "price": "",
    "error": false
  },
  {
    "name": "iPhone 16 Pro Max",
    "price": "",
    "error": false
  },
  {
    "index": 5,
    "error": true,
    "tags": [
      "error"
    ],
    "content": "'str' object has no attribute 'choices'"
  },
  {
    "index": 1,
    "error": true,
    "tags": [
      "error"
    ],
    "content": "'list' object has no attribute 'usage'"
  },
  {
    "index": 2,
    "error": true,
    "tags": [
      "error"
    ],
    "content": "'list' object has no attribute 'usage'"
  }
]